### PR TITLE
Extract two methods from Installer#run

### PR DESCRIPTION
**Pull request by fotanus**
Code climate points it as complexity 384.

This build should fail because the master build is failing. I ran the
specs against ruby 2.2.2 only, so don't merge without pass on travis.

### CODE REVIEW

File: lib/bundler/installer.rb [hunk 1]
```diff
@@ -63,40 +63,14 @@ def run(options)
...
-      # that said, it's a rare situation (other than rake), and parallel
-      # installation is just SO MUCH FASTER. so we let people opt in.
-      jobs = [Bundler.settings[:jobs].to_i-1, 1].max
-      if jobs > 1 && can_install_in_parallel?
-        require 'bundler/installer/parallel_installer'
-        install_in_parallel jobs, options[:standalone], force
-      else
-        install_sequentially options[:standalone], force
-      end
+      resolve_if_need(options)
```
Comment (segiddins) [inline lib/bundler/installer.rb]: `resolve_if_needed`?

File: lib/bundler/installer.rb [hunk 2]
```diff
@@ -63,40 +63,14 @@ def run(options)
...
+      install(options)
 
       lock unless Bundler.settings[:frozen]
       generate_standalone(options[:standalone]) if options[:standalone]
     end
 
+
```
Comment (segiddins) [inline lib/bundler/installer.rb]: Extraneous newline

**Comment by fotanus** [comment]
github questions:
1) Can I `commit --amend` and `push -f` on my branch to fix the issues without having to send other PR and keeping only one commit?
2) Is it desirable?

**Comment by segiddins** [comment]
Yes and yes

-Samuel E. Giddins

On May 17, 2015, at 11:02 AM, Felipe Tanus <notifications@github.com> wrote:

github questions:
1) Can I commit --amend and push -f on my branch to fix the issues without having to send other PR and keeping only one commit?
2) Is it desirable?

**Comment by fotanus** [comment]
Cool feature! Done, thanks!

**Comment by segiddins** [comment]
:+1:

**Comment by indirect** [comment]
Extracting methods is a great improvement! I can't merge this until all the tests pass, though. :)

**Comment by fotanus** [comment]
Of course you can't, otherwise you would be a terrible manager! :-D
I merged updated my master with upstream and it seems it fixed the problem. Lets see what travis says...

**Comment by indirect** [comment]
All good now, thanks!

**Pull request merged**